modules/apps/journal/journal-service/src/test/java/com/liferay/journal/internal/util/JournalConverterImplTest.java

```diff
@@ -88,73 +96,198 @@ private void _testUpdateContentDynamicElement(
 		Assert.assertEquals(expectedValue, rootElement.getStringValue());
 	}
 
-	private void _testUpdateContentDynamicElementWithOptions() {
+	private void _testUpdateContentDynamicElementWithCheckBox() {
 		JournalConverterImpl journalConverterImpl = new JournalConverterImpl();
 
 		ReflectionTestUtil.setFieldValue(
 			journalConverterImpl, "_jsonFactory", new JSONFactoryImpl());
 
-		DDMFormField ddmFormField = _createDDMFormField(
-			"string", true, "field2", "select");
+		SAXReaderImpl saxReaderImpl = new SAXReaderImpl();
 
-		ddmFormField.setMultiple(true);
+		Document document = saxReaderImpl.createDocument();
 
-		DDMFormFieldOptions ddmFormFieldOptions = new DDMFormFieldOptions();
+		Element rootElement = document.addElement("root");
 
-		String value = RandomTestUtil.randomString();
+		boolean[] multipleFlags = {true, false};
+		List<List<String>> allOptionReferences = new ArrayList<>();
+
+		for (boolean multiple : multipleFlags) {
```

Comment:
Why don't you extract this for in a method (named _updateContentDynamicElement, for example) and callit twice with both values?